packages/sw-routing/src/lib/express-route.js

```diff
@@ -25,29 +25,37 @@ import pathToRegExp from 'path-to-regexp';
  * library to transform the `path` parameter into a regular expression, which is
  * then matched against the URL's path.
  *
- * Please note that `ExpressRoute` can only match requests for URLs that are on
- * the same-origin as the current page. If you need to match cross-origin
- * requests, you can use either a generic [`Route`]{@link Route} or a
- * [`RegExpRoute`]{@link RegExpRoute}.
+ * Please note that `ExpressRoute` can match either same-origin or cross-origin
+ * requests. To match only same-origin requests, use a `path` value that begins
+ * with `'/'`, e.g. `'/path/to/(.*)'`. To match cross-origin requests, use
```

Comment:
If you prefer that, I'll switch back. (Those wildcards are valid in Express routes, though, so I figured including some examples using them wouldn't hurt.)